.github/workflows/preview-on-pr-update.yml

```diff
@@ -0,0 +1,69 @@
+name: preview_on_pr_update
```

Comment:
File name is not consistent with `github-ci`

Reply:
this name I took from `boilerplate-mean`, `github-ci` has name `ci` only but that will not reflect the jobs in this file.

Reply:
I'm referring to the file name here, suggesting - `preview_on_pr_update.yml` instead of `preview-on-pr-update.yml`